src/blog/2025/05/designing-flexible-cron-schedules-in-flowfuse-with-node-red.md

```diff
@@ -0,0 +1,333 @@
+{% endrenderFlow %}
+
+If you are willing to see all of the dynamic schedules of the specific node, double-click on it and click on the 'Dynamic Schedules' button from the configuration panel. This will show all of the dynamic schedules associated with that node.
```

Comment:
```suggestion
If you are need to see all of the dynamic schedules of the specific node, double-click on it and click on the 'Dynamic Schedules' button from the configuration panel. This will show all of the dynamic schedules associated with that node.
```